docs-src/python/how-to-use-the-python-sdk-patching-api-in-python.md

```diff
@@ -0,0 +1,45 @@
+---
+id: how-to-use-the-python-sdk-patching-api-in-python
+title: How to use the Python SDK Patching API
+sidebar_label: Python SDK Patching API
+description: Heres a sample implementation of patching in new code using the Python SDK's patching API.
+tags:
+- version
+- python sdk
+- code sample
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/edu-1171/version_your_workflows/workflow_1_initial_dacx.py. -->
+
+In principle, the Python SDK's patching mechanism operates similarly to other SDKs in a "feature-flag" fashion. However, the "versioning" API now uses the concept of "patching in" code.
+
+To understand this, you can break it down into three steps, which reflect three stages of migration:
+
+- Running `pre_patch_activity` code while concurrently patching in `post_patch_activity`.
+- Running `post_patch_activity` code with deprecation markers for `my-patch` patches.
+- Running only the `post_patch_activity` code.
+
+Let's walk through this process in sequence.
```

Comment:
A bit confused on how these files are chopped up. This mentions walking through something in sequence but is its own file and there is no walking through anything here. Is this statement just for the combined page and not this standalone one?

Reply:
> However, the results (the rendered view), makes this phrase make sense.

To confirm, there is no rendered form of this page standalone? I remember an older version of docs let you see these pages standalone _or_ combined. But yeah, if only combined, works for me.